recipes/napari-nd-annotator/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "napari-nd-annotator" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-nD-annotator-{{ version }}.tar.gz
+  sha256: fdb55d27848e876e8f27dde7a1f6b6cc15177e8b2a1764ed21fe1242f62da43f
+  patches:
+    - imports.patch  # Fix opencv import
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - magicgui
+    - matplotlib-base
+    - napari ==0.4.15
+    - numpy
+    - opencv
+    - python >=3.8
+    - qtpy
+    - vispy ==0.9.6
```

Comment:
Can we relax this requirement a bit @bauerdavid? Maybe `>=0.9.6`?

Reply:
Actually I fixed it to 0.9.6 because of this napari issue: https://github.com/napari/napari/issues/4415 (I also experienced this problem). Basically Label rendering in 3D breaks when using 0.10.0. Maybe newer versions of napari or vispy will resolve this problem, but this should be tested.

Reply:
I recommend leaving the pinning to napari.

Reply:
I think the fix was applied for napari==0.4.16, at least it didn't work for me with 0.4.15. This means that the plugin should be ported to 0.4.16. I will do it asap.

Reply:
`napari 0.4.15` was [fixed to pin vispy to `0.9.6`](https://github.com/conda-forge/napari-feedstock/pull/40/files), so we don't need to do it.

Reply:
Okay, I get it now. Thanks!